jaeger-core/src/main/java/io/jaegertracing/internal/metrics/Metrics.java

```diff
@@ -205,4 +205,25 @@ public static String addTagsToMetricName(String name, Map<String, String> tags)
   @Metric(name = "baggage_restrictions_updates", tags = @Tag(key = "result", value = "err"))
   // Number of times baggage restrictions failed to update.
   public Counter baggageRestrictionsUpdateFailure;
+
+  @Metric(name = "deferred_spans_pending")
+  // Number of deferred spans currently pending.
+  public Gauge deferredSpansPending;
+
+  @Metric(name = "filtered_spans")
+  // Number of spans dropped due to short duration.
+  public Counter filteredSpans;
+
+  @Metric(name = "deferred_spans_started")
+  // Number of spans that have been deferred.
+  public Counter deferredSpansStarted;
+
+  @Metric(name = "deferred_spans_finished", tags = @Tag(key = "disposition", value = "dropped"))
```

Comment:
For my knowledge, what do the key and value indicate in `@Tag`

Reply:
It allows you to group counts for the same base metric... `deferred_spans_finished` will contain the total count but you can also query/filter for those where `disposition=dropped` or `disposition=sent`. You can take a look at some of the other preexisting metrics in this file to see how they use `@Tag` as well.